### ISSUE TITLE
gulp-util is missing on npm version of gulp-data-uri

### ISSUE THREAD

**Issue by coderaiser** [versoul/gulp-data-uri#1]
Got an error when try to use **gulp-data-uri**.

```
module.js:340
    throw err;
          ^
Error: Cannot find module 'gulp-util'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (C:\project\some\node_modules\gulp-data-uri\index.j
s:5:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
```

Same thing with **clone-stats**, **strip-ansi**, **has-color** and **ansi-styles**.

**Comment by versoul** [comment]
try now i think fix it in 0.1.1
and answer me work or no please

**Comment by coderaiser** [comment]
everything works good, thank you :)
